tests/modules/submissions/test_tus_creation.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+import os
+import shutil
+
+
+def test_create_submission_from_tus(db, researcher_1):
+
+    from app.modules.submissions.models import Submission
+    from tests.modules.sightings.resources.utils import (
+        prep_tus_dir,
+        get_transaction_id,
+    )  # recycle
+
+    tid = get_transaction_id()  # '11111111-1111-1111-1111-111111111111'
+
+    # first no such dir exists
+    try:
+        sub = Submission.create_submission_from_tus('PYTEST', researcher_1, tid)
+    except OSError:  # this is expected failure behavior here
```

Comment:
So surely the test should ensure it does happen , see 
github.com/WildMeOrg/houston/blob/master/tests/modules/users/test_permissions.py#L12
That fails unless it gets the error.

Reply:
pytest.raises() noted!   changed.